examples/udfilters.c

```diff
@@ -0,0 +1,171 @@
+/*
+  Copyright (C) 2015  Francesc Alted
```

Comment:
Copyright should be:

Copyright (C) 2021  The Blosc Developers

Reply:
Fixed